contentcuration/contentcuration/frontend/channelEdit/views/template/MoveModal.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <v-container grid-list-md>
+    <NewTopicModal
+      :showDialog="showNewTopicModal"
+      @createTopic="createTopic"
+      @cancelTopic="cancelTopic()"
+    />
+
+    <v-layout wrap>
+      <VDialog v-model="showDialog" fullscreen width="500" persistent>
+        <v-toolbar dark color="primary">
+          <VBtn icon dark @click="$emit('cancelMove')">
+            <VIcon>close</VIcon>
```

Comment:
Can use `<Icon>` instead to take advantage of its built-in `notranslate` class (important for auto translations)